assets/javascripts/redmine_issue_history_filter.js

```diff
@@ -186,4 +186,4 @@ var redmineIssueHistoryPlugin = {
 
 redmineIssueHistoryPlugin.process();
 
-window.redmineIssueHistoryPlugin = redmineIssueHistoryPlugin;
```

Comment:
Am I correct to assume that you have removed trailing newline? If I am, then please revert changes to this line.